### PR TITLE
102679 ALW changes 

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description
changes to match figma design  

List of proposed changes:
- as above. 

### What to test for/How to test
- Verify on the [dynamic url](https://ep-be-dyna-102679-alw.bdm-dev-rhp.dts-stn.com/)

### CODE REVIEW

**Comment by alex-solo** [comment]
When estimate = 0, I think the "Next steps" section is missing some info. 
Actual: 
image

Expected:
image

File: utils/api/benefits/alwBenefit.ts [hunk 1]
```diff
@@ -200,6 +219,20 @@ export class AlwBenefit extends BaseBenefit<EntitlementResultGeneric> {
     return { result: formulaResult, type, autoEnrollment }
   }
 
+  /**
+   * Just the formula to get the amount
+   */
+  protected formulaResult(): number {
```
Comment (alex-solo) [inline utils/api/benefits/alwBenefit.ts:220]: If you extracted the formulaResult here, you should use it on line 207.

**Review by alex-solo — DISMISSED** [review]

File: utils/api/benefits/alwBenefit.ts (hunk 1 above)
Comment (MarcoGoC) [reply utils/api/benefits/alwBenefit.ts:220]: done

File: i18n/api/fr.ts [hunk 2]
```diff
@@ -294,10 +294,12 @@ const fr: Translations = {
   },
   detail: {
     eligible: 'Vous êtes probablement admissible à cette prestation.',
+    eligibleIncomeTooHigh:
+      'Vous êtes probablement admissible à cette prestation, mais votre revenu est trop élevé pour recevoir un paiement mensuel pour le moment.',
     eligibleDependingOnIncome:
       'Vous êtes probablement éligible à cette prestation si {INCOME_SINGLE_OR_COMBINED} est inférieur à {INCOME_LESS_THAN}. En fonction de vos revenus, vous devriez vous attendre à recevoir environ {ENTITLEMENT_AMOUNT_FOR_BENEFIT} par mois.',
     eligibleDependingOnIncomeNoEntitlement:
-      "Vous êtes probablement éligible à cette prestation si {INCOME_SINGLE_OR_COMBINED} est inférieur à {INCOME_LESS_THAN}. Une estimation des droits n'est pas disponible à moins que vous ne fournissiez votre revenu.",
+      'Vous êtes probablement admissible à cette prestation si {INCOME_SINGLE_OR_COMBINED} est moins que {INCOME_LESS_THAN}. Fournissez vos revenus pour obtenir une estimation de paiement mensuel.',
```
Comment (alex-solo) [inline i18n/api/fr.ts:302]: image

Need to use this value and replace with amount.

**Review by alex-solo — CHANGES_REQUESTED** [review]

**Review by alex-solo — APPROVED** [review]

**Pull request merged**